Classes/Transformer/ResourceTransformerInterface.php

```diff
@@ -12,11 +12,11 @@
  */
 interface ResourceTransformerInterface
```

Comment:
Not sure if we still need this Interface, as League\Fractal\TransformerAbstract now has type hints and return types (which was the reason for the tests breaking).